lib/beaker/host/unix.rb

```diff
@@ -64,7 +64,7 @@ def self.aio_defaults
       h = self.foss_defaults
       h['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
       h['puppetservice']  = 'puppetserver'
-      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
+      h['puppetbindir']   = '/opt/puppetlabs/agent/bin:/opt/puppetlabs/bin'
```

Comment:
currently the puppet-agent package doesn't symlink files into /opt/puppetlabs/bin. Also many puppet acceptance tests that execute on Unix assume puppetbindir is a single directory (windows doesn't have this issue). So I think if we want to unblock testing, this should just be changed to `h['puppetbindir']   = '/opt/puppetlabs/puppet/bin`